testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -155,6 +155,27 @@ impl BlockMinerThread {
         let mut stackerdbs = StackerDBs::connect(&self.config.get_stacker_db_file_path(), true)
             .expect("FATAL: failed to connect to stacker DB");
 
+        let sort_db = SortitionDB::open(
+            &self.config.get_burn_db_file_path(),
+            true,
+            self.burnchain.pox_constants.clone(),
+        )
+        .expect("FATAL: could not open sortition DB");
```

Comment:
Let's not make any of these calls fatal.  The miner code is constantly re-trying to build blocks, so anything the miner does should be treated as fallible.